project1/index.html

```diff
@@ -0,0 +1,142 @@
+                sunglasses and a backward baseball cap. He had a winter beard and a neon-green cast on his left arm.
+                He’d
+                broken his
+                arm playing hockey with his sons a week before. The first thing Jeff said was, <i>“We’ll head back to
+                    camp,
+                    but I hope you don’t mind we run by the liquor store
+                    first.”</i> I felt more optimistic about my suitability for science.</p>
+        </section>
+
+
+        <div>
+            &bull; &bull; &bull;
+            <!--not sure if this is correct, but I wanted to put these as a separator of the sections and I am having trouble
+        making the spacing above and below to be equal-->
+        </div>
+
+        <section class="last">
+            <!--Here, I was trying to figure out how ti make the bottom margin of the last sentence match the margin in h1-->
+            <p>Not long before I’d called off my engagement it was Christmas.</p>
+
+            <!--Here also, basically all over, I've been trying to clean it up myself, but it doesn't really appear to help much!-->
+
+            <p>The woman who was <i>supposed</i> to be my mother-in-law was a wildly talented quilter and made stockings
+                with Beatrix Potter characters on them for every family member. The previous Christmas she had asked me
+                what
+                character I wanted to be (my fiancé was Benjamin Bunny). I agonized over the decision. It felt
+                important, like whichever character I chose would represent my role in this new family. I chose Squirrel
+                Nutkin, a squirrel
+                with a blazing red tail—an epic, adventuresome figure who ultimately loses his tail as the price for his
+                daring and
+                pride.</p>
+
+            <p>I arrived in Ohio that Christmas and looked to the banister to see where my squirrel had found his place.
+                Instead, I found a mouse. A <em>mouse</em> in a pink dress and apron. A <em>mouse</em> holding a broom
+                and
+                dustpan, serious about sweeping. A<em>mouse</em> named Hunca Munca. The woman who was <em>supposed</em>
+                to
+                become my
+                mother-in-law said, <i>“I was going to do the squirrel but then I thought, that just isn’t CJ. This is
+                    CJ.”</i></p>
+
+            <p>What she was offering was so nice. She was so nice. I thanked her and felt ungrateful for having wanted a
+                stocking, but not this stocking. Who was I to be choosy? To say that this nice thing she was offering
+                wasn’t
+                a thing I wanted?</p>
+
+            <p>When I looked at that mouse with her broom, I wondered which one of us was wrong about who
+                <strong><em>I</em></strong> was.
```

Comment:
Adding a suggestion to keep this in line with the blockquote version:
```suggestion
                <em>I</em> was.
```